<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataMenu</class>
 <widget class="QWidget" name="DataMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>748</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>cmd test</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 211, 211);
QWidget{border-top-left-radius:15px;border-top-right-radius:5px;}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
stop:0#D3D3D3, stop: 1 #D3D3D3);
border:2px solid gray;
border-radius:5px;
margin-top:1ex;/* leave space at the top for the title */
}

QGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top left;
padding: 0 5px;
}</string>
   </property>
   <property name="title">
    <string>Port Config</string>
   </property>
   <widget class="QPushButton" name="pushButtonUsartStart">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 24pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rateCbx_plc">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="portCbx_plc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>651</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Agency FB&quot;;
font: 14pt &quot;Consolas&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Agency FB'; font-size:18pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>421</width>
     <height>711</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Other Config</string>
   </property>
   <widget class="QPushButton" name="pushButton_GetDevId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gen Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>ROBOT CMD TEST</string>
    </property>
    <widget class="QPushButton" name="pushButton_home">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
     </property>
     <property name="text">
      <string>pos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_boat">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_col">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_row">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>220</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Boat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>250</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Col</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>280</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Row</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>50</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>DUT TEST(E2ROM 24c02)</string>
    </property>
    <widget class="QPushButton" name="pushButton_readconfig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
     </property>
     <property name="text">
      <string>read config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_writeconfig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
     </property>
     <property name="text">
      <string>write config</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Address(Hex)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_value_write">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Value(Hex)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_addr_read">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_addr_write">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Address(Hex)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_writeconfig_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
     </property>
     <property name="text">
      <string>AUTO W 0-255</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>390</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelCurRowNumT">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>510</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Agency FB&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>DEMO </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fwd">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>FWD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rev">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>REV</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FRQ">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>520</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Frq</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SC">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>490</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Step Cnt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_moto">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>650</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lock">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>LOCK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unlock">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 18pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>UNLOCK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
